### DISCUSSION TITLE
How to convert excel data to json data?

### DISCUSSION THREAD

**Discussion by superdev1218999** [community/community#49793] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I need to convert data from excel file to json file.
If you know the solution, help me please.
:)

**Comment by 03timothyash** [comment]
Esta bien casi todo es hito @
Nada. Más que ami me  paso que me re valió mis portabilidades y mis antiguos empleos en LinkedIn \ y nunca cobre más del bonus por años

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by 03timothyash** [comment]
Podemos hacer algo para el panel de control remoto de este tipo de contraseña.  Pero solo es necesario tener una conexión de acceso base de datos informáticos

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: Lovely Panda ***@***.***>
Sent: Saturday, March 11, 2023 7:50:48 AM
To: community/community ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GitHub Community] How to convert excel data to json data? (Discussion #49793)


Select Topic Area

Product Feedback

Body

I need to convert data from excel file to json file.
If you know the solution, help me please.
:)
